src/api/v4/leaflet/layer-group.js

```diff
@@ -42,14 +44,20 @@ LayerGroup.prototype._onFeatureClick = function (internalEvent) {
 
 LayerGroup.prototype._onFeatureOver = function (internalEvent) {
   var layer = this._layers.findById(internalEvent.layer.id);
-  if (layer && (layer.hasFeatureClickColumns() || layer.hasFeatureOverColumns())) {
+  if (isInteractive(layer)) {
+    this._hoveredLayers[internalEvent.layerIndex] = true;
```

Comment:
Working with indexes is a bit dangerous. Imagine a layer is removed...

Reply:
Each element has always an unique id